src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -213,20 +215,22 @@ trait ClusterComponent extends LeoComponent {
         .update(Timestamp.from(Instant.now()), ClusterStatus.Deleted.toString, None)
     }
 
-    def setToRunning(googleId: UUID, hostIp: IP): DBIO[Int] = {
+    def updateClusterStatusAndHostIp(googleId: UUID, status: ClusterStatus, hostIp: Option[IP] = None): DBIO[Int] = {
```

Comment:
Maybe remove the default value for `hostIp`, so callers of this method need to explicitly specify it.